.rubocop.yml

```diff
@@ -97,6 +97,7 @@ Metrics/BlockLength:
     - '**/*.gemspec'
 
 Metrics/ModuleLength:
+  Max: 114
```

Comment:
That looks a bit random and kind of unrelated to the rest of the commit. :-)

Reply:
Fair enough, it's the length of the current `MethodDispatchNode` module. Should I disable the cop for the module and revert the config?